Update class.ilUseRootConfirmed.php

ingoj opened the pull request:
added further explanation to file access rights if setup is executed as root.

ingoj committed:
Update class.ilUseRootConfirmed.php

added further explanation to file access rights

klees reviewed (changes requested):
Hi @ingoj,

thanks a lot and nice to see a contribution from you. One minor change request:

* [x] Please end the additional sentence with a full stop.

Thanks!

ingoj committed:
Update class.ilUseRootConfirmed.php

Correct Typo

ingoj committed:
Update README.md

Added hint for filesystem permissions, if setup is not executed by the user that runs the webserver

ingoj committed:
Merge branch 'ILIAS-eLearning:release_7' into ingoj-setup-1

klees commented:
Thank you =)
